packages/peregrine/src/validators/iterable.js

```diff
@@ -0,0 +1,25 @@
+const isIterable = obj => typeof obj[Symbol.iterator] === 'function';
+
+function optionalIterable(props, propName, componentName) {
+    const prop = props[propName];
+
+    if (prop != null && !isIterable(prop)) {
+        return new Error(
+            `Invalid prop \`${propName}\` supplied to \`${componentName}\`. Validation failed.`
```

Comment:
Can we make this more specific? Doesn't seem to call out that we're expecting an iterable

Reply:
@DrewML Updated. 👍